packages/bot-web-ui/src/components/dashboard/quick-strategy/quick-strategy-components/fields.tsx

```diff
@@ -0,0 +1,228 @@
+                                        is_uniq_strategy_field={is_uniq_strategy_field}
+                                        trailing_icon_message={trailing_icon_message}
+                                        zIndex={zIndex}
+                                        uniq_selected_input={uniq_selected_input}
+                                        errors={errors}
+                                    />
+                                    <InputField
+                                        idx={idx}
+                                        handleChange={handleChange}
+                                        onChangeInputValue={onChangeInputValue}
+                                        setCurrentFocus={setCurrentFocus}
+                                        is_mobile={is_mobile}
+                                        errors={errors}
+                                    />
+                                </div>
+                            );
+                        }
+                        return (
+                            <div key={idx}>
+                                <div className='quick-strategy__form-row'>
+                                    <SelectField
+                                        field_name={field_name as TSelectsFieldNames}
+                                        id={id}
+                                        is_mobile={is_mobile}
+                                        dropdown_list={dropdown_list}
+                                        selected_value={selected_value}
+                                        label={label}
+                                        select_value={select_value as TDropdownItems}
+                                        setFieldValue={setFieldValue}
+                                        className={className}
+                                        is_able_disabled={is_able_disabled}
+                                        values={values}
+                                        onChangeDropdownItem={onChangeDropdownItem}
+                                        onHideDropdownList={onHideDropdownList}
+                                        onScrollStopDropdownList={onScrollStopDropdownList}
+                                        selected_trade_type={selected_trade_type}
+                                        selected_symbol={selected_symbol}
+                                    />
+                                </div>
+                                {id.endsWith('types-strategies') && (
+                                    <div key='description' className='quick-strategy__description'>
+                                        {description}
+                                    </div>
+                                )}
+                            </div>
+                        );
+                    }
+                    return false;
+                }),
+            // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
can you please remove this comment? 
if there is some issues with the `deps` array, it's better we fix them now.

Reply:
usually it breaks vercel deployments if we don't add this, will check on it 🙇

Reply:
@michio-rework in the current case, we really need an empty dependency to do it only on the first render but to avoid a warning I added this comment